apps/bfd-server/bfd-server-launcher/src/test/java/gov/cms/bfd/server/launcher/ServerProcess.java

```diff
@@ -179,27 +178,22 @@ static ProcessBuilder createAppProcessBuilder(Path warPath, JvmDebugOptions jvmD
   /**
    * @param jvmDebugOptions the {@link JvmDebugOptions} to use
    * @return the command array for {@link ProcessBuilder#ProcessBuilder(String...)} that will launch
-   *     the application via its <code>.x</code> capsule executable
+   *     the application via its <code>.x</code> executable wrapper script
    */
-  static String[] createCommandForCapsule(JvmDebugOptions jvmDebugOptions) {
+  static String[] createCommandForLauncher(JvmDebugOptions jvmDebugOptions) {
     try {
-      Path javaBinDir = Paths.get(System.getProperty("java.home")).resolve("bin");
-      Path javaBin = javaBinDir.resolve("java");
-
-      Path buildTargetDir = Paths.get(".", "target");
-      Path appJar =
-          Files.list(buildTargetDir)
+      Path assemblyDirectory =
+          Files.list(Paths.get(".", "target", "server-work"))
               .filter(f -> f.getFileName().toString().startsWith("bfd-server-launcher-"))
-              .filter(f -> f.getFileName().toString().endsWith("-capsule-fat.jar"))
+              .findFirst()
+              .get();
+      Path serverLauncherScript =
+          Files.list(assemblyDirectory)
+              .filter(f -> f.getFileName().toString().equals("bfd-server-launcher.sh"))
               .findFirst()
               .get();
```

Comment:
Same comment as previous file.  Also, why does this exist twice?

Reply:
it exists once for the pipeline and once for the server. agree with the original comment -- it was carryover from using the version number in the script name and it no longer necessary